tst/testinstall/recordname.tst

```diff
@@ -71,4 +71,97 @@ Syntax error: Identifiers in GAP must consist of at most 1023 characters. in s\
 tream:2
 aaaxyz := 1; # 1024 chars
  ^^^^^
+gap> r := rec(y := 2);
```

Comment:
I would suggest to separate the new tests from the existing ones; i find that this makes `.tst` files a lot easier to read.
```suggestion

# tests for issue #3489 
gap> r := rec(y := 2);
```